qemu/tests/balloon_hotplug.py

```diff
@@ -66,6 +65,10 @@ def run_pm_test(pm_test, plug_type):
             err += ("\nDevice is not in qtree %ss after hotplug:\n%s"
                     % (pause, vm.monitor.info("qtree")))
 
+        vm.params["balloon"] = "balloon%d" % idx
+        vm.params["balloon_dev_devid"] = "balloon%d" % idx
```

Comment:
seems idx is always 0 according to line42, could we replace idx with 0 directly?

Reply:
Hard coding is not a good choice, thanks!